proto/v1/activity_service.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package bytebase.v1.activity;
```

Comment:
Shouldn't be activity service, let's inline with project service for now.

Reply:
Done, move the `message activity` to project service file for now.